backend/pkg/api/connect/integration/api_suite_test.go

```diff
@@ -53,29 +63,59 @@ func (s *APISuite) SetupSuite() {
 	t := s.T()
 	require := require.New(t)
 
-	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
 	defer cancel()
 
-	// 1. Start Redpanda Docker container
-	container, err := redpanda.RunContainer(ctx, testcontainers.WithImage("redpandadata/redpanda:v23.2.18"))
-	container.FollowOutput(testutil.TestContainersLogger{LogPrefix: "CONTAINER (connect-integration): "})
+	// 1. setup docker network
+	networkName := "redpanda-net"
+	network, err := setupNetwork(networkName)
+
+	require.NoError(err)
+
+	s.network = network
+
+	// 2. Start Redpanda Docker container
+	container, err := redpanda.RunContainer(ctx, testcontainers.WithImage("redpandadata/redpanda:v23.2.18"), testcontainers.WithNetwork(networkName, "redpanda"))
+	container.FollowOutput(testutil.TestContainersLogger{LogPrefix: "CONTAINER REDPANDA (connect-integration): "})
 	require.NoError(container.StartLogProducer(context.Background()))
 
+	if err != nil {
+		t.Fatal(err)
+	}
+
 	require.NoError(err)
 	s.redpandaContainer = container
 
-	// 2. Retrieve connection details
+	// 2. Retrieve Redpanda connection details
 	seedBroker, err := container.KafkaSeedBroker(ctx)
 	require.NoError(err)
 	schemaRegistryAddress, err := container.SchemaRegistryAddress(ctx)
 	require.NoError(err)
 
+	dockerSeedBroker, err := container.DockerKafkaSeedBroker(ctx)
+
+	require.NoError(err)
+
 	s.testSeedBroker = seedBroker
 
-	// 3. Create Kafka clients
+	// 3. Start Kafka Connect Docker container
+	kConnectContainer, err := runKafkaConnectContainer(s.network.Name, []string{dockerSeedBroker})
+	kConnectContainer.FollowOutput(testutil.TestContainersLogger{LogPrefix: "CONTAINER CONNECT (connect-integration): "})
+	require.NoError(kConnectContainer.StartLogProducer(context.Background()))
+
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Same as above: I think we should use require.NoError() closer to where the error is returned instead